_posts/en/newsletters/2025-03-28-newsletter.md

```diff
@@ -0,0 +1,293 @@
+
+- [LND 0.19.0-beta.rc1][] is a release candidate for this popular LN
+  node.  One of the major improvements that could probably use testing
+  is the new RBF-based fee bumping for cooperative closes described
+  below in the notable code changes section.
+
+<!-- FIXME:harding to update Thursday -->
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+- [Bitcoin Core #31603][] updates the `ParsePubkeyInner` parser to reject public
+  keys with leading or trailing whitespace, matching the parsing behavior of the
+  [rust-miniscript][rust miniscript] project.  It shouldn't have been
+  possible to accidentally add whitespace previously due to the
+  protection of the descriptor checksum. The `getdescriptorinfo` and
+  `importdescriptors` RPC commands now throw an error if the public key fragment
+  of a [descriptor][topic descriptors] contains such whitespace.
+
+- [Eclair #3044][] increases the default minimum confirmations for channel
+  safety against block reorganizations from 6 to 8.  It also removes the scaling of
+  this value based on the channel funding amount because channel capacity can be changed
+  significantly during [splicing][topic splicing], convincing the node
+  to accept a low number of confirmations for what is actually a large
+  amount of money at stake.
+
+- [Eclair #3026][] adds support for Bitcoin Core wallets using [Pay-to-Taproot
+  (P2TR)][topic taproot] addresses, including watch-only wallets managed by
+  Eclair, as a basis for implementing [simple taproot channels][topic simple
+  taproot channels]. P2WPKH scripts are still required for some mutual close
+  transactions, even when using a P2TR wallet.
+
+- [LDK #3649][] adds support for paying Lightning Service Providers (LSPs) with
+  [BOLT12 offers][topic offers] by adding the necessary fields. Previously, only
+  [BOLT11][] and on-chain payment options were enabled. This was also proposed
+  in [BLIPs #59][].
+
+- [LDK #3665][] increases the [BOLT11][] invoice size limit from 1,023 bytes to
+  7,089 bytes to match LND's limit, which is based on the maximum number
+  of bytes that can fit on a QR code.  The PR author argues that QR
+  codes compatible with the encoding used in a BOLT11 invoice are
+  actually limited to 4,296 characters, but the 7,089 value is chosen
+  for LDK because "system-wide consistency is probably more important."
+
+- [LND #8453][], [#9559][lnd #9559], [#9575][lnd #9575], [#9568][lnd
```

Comment:
8453 was closed? @harding @Gustavojfe

Reply:
Yeah, but it was where the work originally occurred, so I think it's worth mentioning.  I listed the PRs in numeric order, so it (somewhat unfortunately) comes first.